source/codegen/metadata/nifake/functions.py

```diff
@@ -1240,6 +1240,41 @@
         ],
         'returns': 'ViStatus'
     },
+    'GetPatternPinIndexes':{
```

Comment:
Can this function name be more generic? Currently I see one metadata called as `GetAnIviDanceWithATwistString`. Can something similar be used for this so that its easy to read

Reply:
Yeah, I could change it. I just added it like [this](https://github.com/doshirohan/grpc-device/blob/1081b6caf61c479d3e7dcca7006f3aa788ae377e/source/codegen/metadata/nifake/functions.py#L278) was added for ivi-dance